test/hotspot/jtreg/runtime/condy/CondyExtendedNullPointerTest.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) 2024, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+/*
+ * @test
+ * @bug 8339488
+ * @summary Test extended NullPointerException message in method with CONSTANT_Dynamic.
+ * @requires vm.flagless
```

Comment:
Is there are any reasons to ignore all additional vm flags? Usually the test should accept any flags and be flagless only if there are known reasons for this. Even if it doesn't looks to be useful to run in different modes. Usually to stress comiplers with different bycodes.
You don't need to run test in an additional configurations.

Reply:
I copied from CondyLDCTest - I figured the authors must have thought about something I didn't.

Reply:
I quickly looked through test history and didn't find any reason for this. Could you please change
'createLimitedTestJavaProcessBuilder'
to 
'createTestJavaProcessBuilder'
and remove 
line 28
@requires vm.flagless.

I'll file a bug to fix CondyLDCTest also.

Reply:
BTW, I filed PR and you might see that one of tests start crash. So it is really useful to accept vm flags.

Reply:
CondyLDCTest test crashes with VM flags?

Reply:
Sorry, I forgot to add link. Test runtime.condy.TestEscapeThroughInvokeWithCondy fails. See PR 
https://github.com/openjdk/jdk/pull/20894

Reply:
Thanks for looking. Updated as you suggested, @lmesnik, please take a quick look when you can.